pylxd/models/network.py

```diff
@@ -41,14 +50,25 @@ def all(cls, client):
             networks.append(cls(client, name=name))
         return networks
 
+    @classmethod
+    def create(cls, client, name, description=None, type_=None,
+               config=None):
+        """Create a network"""
+        network = {'name': name}
```

Comment:
This is where we'd do a check for the 'network' API extension and bail with an appropriate Exception.

Reply:
I'll add this check.
BTW. Do you have any idea how to replace checking every `__init__` argument for `None` value? In my opinion we could use `**kwargs` instead and check if all keys of this dictionary are valid (using introspection for example).

Reply:
Yeah, this is a quirk of Python; there isn't a nice way to do this that also keeps readability of the function signature.  I'd just keep what you have to be honest.